src/templates/speciality.js

```diff
@@ -27,6 +27,7 @@ const Speciality = ({
   },
   location
 }) => {
+  console.log('speciality', speciality)
```

Comment:
remove

Reply:
has there been any previous discussion of adding no console log rules to eslint?

Reply:
There hasn't been any previous discussion. I thought it was already in there as a rule :confused:

Reply:
I thought so as well. Has it been removed?